docs/_includes/engage.html

```diff
@@ -1,6 +1,6 @@
 <!-- engage -->
 <div class="engage">
-    <section class="page-section" id="{{ site.data.sitetext.engage.section | default: "engage" }}">
+    <section class="newpadding" id="{{ site.data.sitetext.engage.section | default: "engage" }}">
```

Comment:
@Neilblaze can we use the same class name instead of creating a new class? If we need a new class, because we're using "page-section" elsewhere still, can we use "padded-page-section" or something similar?

Reply:
`padded-page-section` seems better to me :thinking: 
Lemme make the quick edit.